corehq/apps/cloudcare/static/cloudcare/js/formplayer/spec/util_spec.js

```diff
@@ -141,5 +141,63 @@ describe('Util', function () {
             assert.equal(lastCall.returnValue.title, "Some Cases");
             assert.deepEqual(lastCall.returnValue.breadcrumbs, ["My App", "Some Cases"]);
         });
+
+        it("should paginate, filter, and sort case lists", function () {
+            FormplayerFrontend.trigger("menu:select", 1);
+            FormplayerFrontend.trigger("menu:perPageLimit", 2);
+            var url = Util.currentUrlToObject();
+            assert.equal(url.casesPerPage, 2);
+            assert.equal(url.page, null);
```

Comment:
I think it's the other way around, we should check for `null` instead of falsiness if we know the value is going to be null. Updated the `isNotOk` tests in 172b358a56d